web/styles/left_sidebar.css

```diff
@@ -714,8 +718,17 @@ li.topic-list-item {
     top: -0.5px;
     z-index: 1;
 
-    & input {
-        padding-right: 20px;
+    .input-append {
```

Comment:
It might be better in this sort of situation to drop `input-append` from the HTML and just use some other semantic name for the specific element that needs these styles.

Reply:
Yes, we should drop `input-append`. Will add this in the next PR.